src/commands/call.ts

```diff
@@ -57,6 +69,42 @@ export class ActorCallCommand extends ApifyCommand<typeof ActorCallCommand> {
             runOpts.memory = this.flags.memory;
         }
 
+        let input: Record<string, unknown> | undefined;
+
+        if (this.flags.input) {
+            switch (this.flags.input[0]) {
+                case '@': {
```

Comment:
@netmilk I don't understand why we need to pass files with the `@` prefix - could you elaborate on the reasoning behind that please?

My first impression is that I wouldn't expect a file argument to require a special syntax.

Reply:
Good catch, @janbuchar. I think Marek's original intention was to distinguish whether it's an input string or a reference to a file, the same way [`curl` does it](https://unix.stackexchange.com/questions/144479/what-does-the-at-symbol-before-a-filename-mean-in-a-curl-command). 

I would prefer to **avoid** doing that. Please use `--input-file` instead as proposed in the soon-to-be-published, but still [internal CLI Design Final Draft's key design principles](https://www.notion.so/apify/API-CLI-design-Final-Draft-517b46afd5b8431dae7e38b06c9080fc?pvs=4#68bf93c034f84e9d80dcb325805fb875).